v2/spanner-change-streams-to-sharded-file-sink/src/main/java/com/google/cloud/teleport/v2/templates/changestream/DataChangeRecordTypeConvertor.java

```diff
@@ -157,6 +157,9 @@ public static BigDecimal toNumericBigDecimal(
       throws DataChangeRecordConvertorException {
 
     String value = toString(changeEvent, key, requiredField);
+    if (value == null) {
+      return null;
+    }
```

Comment:
If there is DataChangeRecordTypeConverter Unit test, can you add a test case to cover these 3 lines?